### ISSUE TITLE
All my changes disappear after docker container restart

### ISSUE THREAD

**Issue by jayakumarj85** [project-flogo/flogo-web#1135]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report => search github for a similar issue or PR before submitting
[ ] feature request
[ ] support request
```

**Current behavior**
I am running flogo using the docker as per the documentation and created a new App and made modifications to the app by adding flows to it. I was able to export the binary and test it successfully without any issues. One thing I forgot to do is to export the App that I have created as a json file. I then stopped my docker container and started the same container the next day and found that the App along with the flows that I have created are gone. Now I could see only a sample app when I open my Flogo Web UI. This appers to be an issue everytime I make changes and restart my docker container.

**Expected behavior**
All the App & flows that I have created using Flogo Web UI must be persisted and must be available after restarting the docker container.

**Minimal reproduction of the problem with instructions**
1. Run flogo using `docker run -it -p 3303:3303 flogo/flogo-docker:latest`
2. Open Flogo Web UI and create a new App thereby creating a flow inside the new App.
3. Stop the docker container using `docker stop <container_name>`
4. Start the same docker container using `docker start <container_name>`
5. Open Flogo Web UI and check if the new App created in step 2 is available for modification.

**What is the motivation / use case for changing the behavior?**
All the Apps and flows created must be persisted automatically into the docker container.

**Please tell us about your environment:**
OS - Windows 10
Docker - 2.0.0.3 (31259)
Node - v10.15.3

* **Flogo version:** 0.X.X
Flogo v0.9.2

* **Browser:** [all | Chrome XX | Firefox XX | IE XX | Safari XX ]
I have tried it with Mozilla Firefox 68.0

**Additional information you deem important (e.g. issue happens only occasionally):**
Issue happens everytime I restart the docker container

**Comment by apandurangi** [comment]
@fcastill could you please confirm if this ticket is fixed with https://github.com/project-flogo/flogo-web/pull/1133?

**Comment by fcastill** [comment]
it was indeed fixed by #1133 and now the latest `flogo/flogo-docker` image contains the fix
